virttest/utils_test/libvirt.py

```diff
@@ -1425,7 +1425,13 @@ def do_migration(self, vms, srcuri, desturi, migration_type,
                 if func:
                     # Execute command once the migration is started
                     migrate_start_state = args.get("migrate_start_state", "paused")
-                    if self.wait_for_migration_start(vm, state=migrate_start_state, uri=desturi):
+
+                    # Wait for migration to start
+                    migration_started = self.wait_for_migration_start(vm, state=migrate_start_state,
+                                                                      uri=desturi,
+                                                                      migrate_options=options+' '+extra_opts)
```

Comment:
```python
if options:
    migrate_options = str(options)
if extra_opts:
    migrate_options += " %s" % extra_opts
migration_started = self.wait_for_migration_start(vm, state=migrate_start_state,
                                                  uri=desturi,
                                                  migrate_options=migrate_options.strip())
```

Reply:
please have this fix, else we could hit,
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

Reply:
Thanks. updated